string zone_name # robot_ns_zone
zone_check[] zone_data